elo7/fastify-svelte-partial-hydration
# [ELO-4610] Add initial files

# :warning: Attention

# Changelog
- Add REAME.md;
- Add `hydrate.js` script;
- Add plugin default export;
- Add Hydrate component

# Status
## Developer
1. [x] Developed
2. [x] Manually tested
3. [x] Documented

## Reviewer
1. [x] Manually tested
2. [x] Documentation reviewed

# Tests
Describe what tests need to be done manually?

- Run `npm link` on this project folder
- Go to the `@elo7/fastify-svelte` project, checkout to the `partial-hydration` branch and run `npm link`
- Go to a project where you will use both plugins and run `npm link @elo7/fastify-svelte @elo7/fastify-svelte-partial-hydration`
- Register the plugin using `fastify-svelte`
- [x] Check if your Svelte templates are rendering correctly and partially
